README.md

```diff
@@ -799,6 +801,89 @@ Edit the `*.auto.pkrvars.hcl` file in each `builds/<type>/<build>` folder to con
   >
   > All `variables.auto.pkrvars.hcl` default to using the [VMware Paravirtual SCSI controller][vmware-pvscsi] and the [VMXNET 3][vmware-vmxnet3] network card device types.
 
+##### Network Variables (Optional)
+
+Edit the `config/network.pkrvars.hcl` file to configure a static IP address:
+
+- IPv4 address, netmask, and gateway.
+- DNS list.
+
+Without any configuration, the network is set using DHCP protocol.
+
+**Example**: `config/network.pkrvars.hcl`
+
+```hcl
+vm_ip_address = "172.16.100.192"
+vm_ip_netmask = 24
+vm_ip_gateway = "172.16.100.1"
+vm_dns_list   = [ "172.16.11.4", "172.16.11.5" ]
+```
+
+> **Note**
+>
+> Static IP assignement is only supported by Linux (except SUSE) for now.
+
+##### Storage Variables (Optional)
+
+Edit the `config/storage.pkrvars.hcl` file to configure a partitioning scheme:
+
+- Disk device and whether to use a swap partition.
+- Disk partitions and related settings.
+- Logical volumes and related settings (optional).
+
+**Example**: `config/storage.pkrvars.hcl`
+
+```hcl
+vm_disk_device = "sda"
+vm_disk_use_swap = false
+vm_disk_partitions = [
+  {
+    name = "efi"
+    size = 1024,
+    format = {
+      label  = "EFIFS",
+      fstype = "fat32",
+    },
+    mount = {
+      path    = "/boot/efi",
+      options = "",
+    },
+    volume_group = "",
+  },
+  {
+    name = "boot"
+    size = 1024,
+    format = {
+      label  = "BOOTFS",
+      fstype = "xfs",
+    },
+    mount = {
+      path    = "/boot",
+      options = "",
+    },
+    volume_group = "",
+  },
+  {
+    name = "root"
+    size = -1,
+    format = {
+      label  = "ROOTFS",
+      fstype = "xfs",
+    },
+    mount = {
+      path    = "/",
+      options = "",
+    },
+    volume_group = "",
+  },
+]
+vm_disk_lvm = []
+```
+
+> **Note**
+>
+> Storage settings is only supported by Linux (except Photon and SUSE for now).
```

Comment:
```suggestion
> Storage settings are only supported for Linux machine images (except Photon OS and SUSE).
```

Reply:
Note: This should be possible in Photon OS 5.0 (Beta.) The installer was updated last year.

Reply:
Done